AnalyticsAdmin/src/V1alpha/AudienceDimensionOrMetricFilter/StringFilter/MatchType.php

```diff
@@ -49,12 +49,6 @@ class MatchType
      * Generated from protobuf enum <code>FULL_REGEXP = 5;</code>
      */
     const FULL_REGEXP = 5;
-    /**
-     * Partial regular expression matches with the string value.
-     *
-     * Generated from protobuf enum <code>PARTIAL_REGEXP = 6;</code>
-     */
-    const PARTIAL_REGEXP = 6;
```

Comment:
this is a breaking change, but as we are in `preview` (pre-1.0), this is acceptable.